content/events/19-20/mini-lan.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 'Geeks, Attack: Bootcamp!'
+description: Zeus' bootcamp LAN-Party!
+created_at: 05-03-2020
+time: '27-03-2020 18:00'
+end: '28-03-2020 4:00'
+facebook: 'https://www.facebook.com/events/648201269331401'
+image: ''
+banner: ''
```

Comment:
If empty, remove them, otherwise it will try and load the image at url `''`, which obviously doesn't exist.